### PR TITLE
UI tweaks for community page

### PR DESCRIPTION
**Pull request by dem4ron**
## Before:

Screenshot 2022-10-25 at 11 44 15

---

Screenshot 2022-10-25 at 11 49 57

## After:

Screenshot 2022-10-25 at 12 15 20

---

Screenshot 2022-10-25 at 11 49 54

## Avatars
Screenshot 2022-10-25 at 12 11 09

### CODE REVIEW

File: app/views/community/stories/show.html.haml [hunk 1]
```diff
@@ -31,22 +31,15 @@
...
-        = graphical_icon "podcast-gradient", css_class: "h-[48px] w-[48px] mb-24"
+        = graphical_icon "podcast-gradient", css_class: "h-[48px] w-[48px] mr-24"
         .flex.flex-col
           %h2.text-h2.mb-8 More Stories from our community
           %p.text-p-large Listen, learn and be inspired by our community members.
       .grid.grid-cols-3.gap-16
         - @other_stories.each do |community_story|
           = link_to community_story_path(community_story), class: 'flex flex-col grid shadow-smZ1 p-16 bg-white rounded-8' do
-            .bg-borderLight.rounded-8.mb-12{ class: "w-[100%] pb-[56.25%]" }
+            = image_tag community_story.thumbnail_url, class: "bg-borderLight rounded-8 mb-12 w-[100%]", width: 243, height: 136
```
Comment (iHiD) [inline app/views/community/stories/show.html.haml]: This isn't actually on `main` but I can't rebase it out, which is confusing me here! 

@dem4ron @ErikSchierboom Any ideas?

File: app/css/pages/community-story.css [hunk 2]
```diff
@@ -9,7 +9,7 @@ body.namespace-community.controller-stories.action-show {
 
     .back-link {
         @apply flex items-center relative py-32;
-        @apply text-textColor6 text-16 font-semibold leading-170;
+        @apply text-lightBlue text-16 font-semibold leading-170;
```
Comment (iHiD) [inline app/css/pages/community-story.css:12]: IIRC think light blue on black violates accessibilty so I changed this. I think it should be even brighter really.

File: app/views/community/stories/show.html.haml (hunk 1 above)
Comment (iHiD) [reply app/views/community/stories/show.html.haml]: I had this wrong - I didn't realise it was duplicated. I've pushed this to main now. Thanks @dem4ron!

**Review by ErikSchierboom — COMMENTED** [review]
Nothing much to review for me here 🤷

File: app/css/pages/community-story.css (hunk 2 above)
Comment (dem4ron) [reply app/css/pages/community-story.css:12]: I'm fine with that. then we can change the arrow's colour too, so it looks neat.

**Comment by dem4ron** [comment]
I'm closing this, as it is basically merged

**Pull request closed without merging**